examples/Boost-stacktrace/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2013, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-boost)
+
+hunter_add_package(Boost COMPONENTS system stacktrace)
+find_package(Boost CONFIG REQUIRED system stacktrace)
```

Comment:
I made all my changes and new tests in the pkg.boost branch.
https://github.com/Dargun/hunter/pull/2
i also merged all stuff now in upload.boost branch
https://github.com/Dargun/hunter/tree/upload.boost

do i merge that then back into ingenue/hunter or ruslo/hunter
it's for me not clear?

Reply:
Example and documentation goes to ruslo/hunter:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests

`.travis.yml`/`appveyor.yml` updates goes to ingenue/hunter@pkg.boost:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests-tests

Since you're adding new CI tests first pull request is ruslo/hunter, then I will merge and update pkg.boost, then you can do second pull request.